backend-tests/tests/test_tenantadm_support.py

```diff
@@ -34,9 +34,6 @@
 
 
 class TestContactSupportEnterprise:
-    @pytest.mark.skipif(
-        isK8S(), reason="not testable in a staging or production environment"
-    )
     def test_contact_support_bad_request(self, clean_mongo):
```

Comment:
Shouldn't we skip also this one?

Reply:
@lluiscampos as it returns "bad request", no email is sent, thus the test can run. Am I wrong?
The reason why we cannot run this test is because the staging env will send real emails, and we won't catch them from the mocked SMTP server.

Reply:
Yeah, I was only wondering if it had value at all. But with your reply I understand it does: it will verify the `tenantadm` logic :+1: